sds_data_manager/utils/stackbuilder.py

```diff
@@ -0,0 +1,35 @@
+"""Module with helper functions for creating standard sets of stacks"""
+# Installed
+from aws_cdk import (
+    App,
+    Environment
+)
+
+# Local
+from sds_data_manager.stacks import (
+    opensearch_stack,
+    sds_data_manager_stack
+)
+
+
+def build_sds(scope: App, env: Environment,
```

Comment:
I really like that there's a separate function for defining the entire stack. I think it reduces confusion a lot!